sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Custom/Models/SecurityAssessmentMetadataData.Serialization.cs

```diff
@@ -189,6 +187,10 @@ internal static SecurityAssessmentMetadataData DeserializeSecurityAssessmentMeta
                                 property0.ThrowNonNullablePropertyIsNull();
                                 continue;
                             }
+                            if (property0.Value.ValueKind == JsonValueKind.String && string.IsNullOrEmpty(property0.Value.GetString()))
```

Comment:
Considering the type of this property in swagger should be `string`, maybe we can simplify the code here:
```suggestion
                            if (string.IsNullOrEmpty(property0.Value.GetString()))
```